spec/factories/country_permissions.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :country_permission do
+    country { 1 }
+    video { 1 }
```

Comment:
```suggestion
    country
    video
```

This way it creates the associated models and does not hardcode an id.